src/http.rs

```diff
@@ -36,10 +43,15 @@ where
         Ok(HTTP {
             client: C::default(),
             app_name,
+            sdk_version: get_sdk_version(),
+            connection_id: Uuid::new_v4().into(),
```

Comment:
I decided to instantiate the UUID when we create our HTTP client since we don't need it anywhere else. Since this is intended for internal tracking purposes of SDK HTTP connection and users should not override it I put it here instead of passing it from the outside.

Reply:
Does this make sense for the Rust SDK as-is? A new UUID every process restart doesn't provide much value. (And indeed for k8s we'd see 10's of thousands of clients a week...

Reply:
We've been using it for 2 weeks already and it provides a lot of value to us on the server side. We want to know how many instances are created. We track it in hourly buckets with hyperloglog so very high numbers are not a problem and they help us in capacity planning and debugging of misconfigured SDK clients (with per request scope objects)